forge-gui/res/cardsfolder/upcoming/sparas_bodyguard.txt

```diff
@@ -0,0 +1,15 @@
+Name:Spara's Bodyguard
+ManaCost:G W U
+Types:Creature Rhino Warrior
+PT:3/3
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigChoose | TriggerDescription$ When CARDNAME enters the battlefield, you may choose a creature card in your hand. If you do, it perpetually gains "This creature enters the battlefield with an additional shield counter on it." Otherwise, put a shield counter on CARDNAME.
+SVar:TrigChoose:DB$ ChooseCard | ChoiceZone$ Hand | Choices$ Creature.YouOwn | ChoiceTitle$ You may choose a creature card in your hand | SubAbility$ DBEffect
+SVar:DBEffect:DB$ Effect | ConditionDefined$ ChosenCard | ConditionPresent$ Card | StaticAbilities$ PerpetualShield | Name$ Spara's Bodyguard's Perpetual Effect | Duration$ Permanent | SubAbility$ DBAddCounter
+SVar:PerpetualShield:Mode$ Continuous | Affected$ Card.ChosenCard | AddKeyword$ etbCounter:SHIELD:1 | EffectZone$ Command | AffectedZone$ Battlefield,Hand,Graveyard,Exile,Stack,Library,Command | Description$ The chosen card perpetually enters the battlefield with an additional shield counter.
```

Comment:
Rather than adding the keyword, do you mind doing this like Master Chef or Jade Orb?

We are trying to get away from using keywords for things that aren't technically keywords.